tests/integ_tests/workflow_tests/continue_as_new.rs

```diff
@@ -27,9 +27,10 @@ async fn continue_as_new_happy_path() {
     let wf_name = "continue_as_new_happy_path";
     let mut starter = CoreWfStarter::new(wf_name);
     let worker = starter.worker().await;
+    worker.register_wf(wf_name.to_string(), continue_as_new_wf);
 
     worker
-        .submit_wf(vec![[1].into()], wf_name.to_owned(), continue_as_new_wf)
+        .submit_wf(wf_name.to_string(), wf_name.to_string(), vec![[1].into()])
         .await
         .unwrap();
```

Comment:
Might be nice to create a combined register & submit convenience function for all these.

Reply:
I was actually thinking of making `register_wf` return a type-safe function that you can call.
If you don't mind I'll open an issue to unblock this PR.

Reply:
Sounds good